### PR TITLE
Dark mode

### PR DESCRIPTION
**Pull request by darshanraju**

### CODE REVIEW

**Comment by darshanraju** [comment]
image

File: frontend/src/components/trainModelForm.tsx [hunk 1]
```diff
@@ -232,6 +222,11 @@ const TrainModelForm = ({ setSendingRequest }: ITrainModelForm) => {
                     required={true}
                     gridSize={6}
                     color="primary"
+                    formControlLabelProps={{
+                      className: classes.formHeaders,
+                      // style: { textDecoration: "red" },
```
Comment (hmalik88) [inline frontend/src/components/trainModelForm.tsx:227]: what's the comment for?

File: frontend/src/components/trainModelForm.tsx [hunk 2]
```diff
@@ -250,10 +245,18 @@ const TrainModelForm = ({ setSendingRequest }: ITrainModelForm) => {
                     value="max_num_epochs"
                     defaultValue="max_num_epochs"
                     radioGroupProps={{ defaultValue: "max_num_epochs" }}
+                    // formLabelProps={{ className: classes.formHeaders }}
```
Comment (hmalik88) [inline frontend/src/components/trainModelForm.tsx:248]: what're these comments for?

File: frontend/src/components/trainModelForm.tsx (hunk 1 above)
Comment (darshanraju) [reply frontend/src/components/trainModelForm.tsx]: removed them, thanks

File: frontend/src/components/trainModelForm.tsx (hunk 2 above)
Comment (darshanraju) [reply frontend/src/components/trainModelForm.tsx]: removed them, thanks

**Review by hmalik88 — APPROVED** [review]
LGTM

**Review by daniniguez — APPROVED** [review]

**Pull request merged**